core/src/wasm/interval.rs

```diff
@@ -67,6 +69,9 @@ pub struct Interval {
 impl Interval {
     /// Create a new `Interval` stream that will resolve each given duration.
     pub fn new(period: Duration) -> Self {
+        if let Err(e) = init_timer_overrides() {
+            workflow_log::log_error!("Unable to initialize timer overrides: {:?}", e);
```

Comment:
Since you changed return type to String, no longer to output using debug... in fact you can just `log_error!(e);`

Reply:
a2f25c0efe6a1120867a044ce3e95715ed5d9804